### PR TITLE
Added  Urdu Language Translation in Form Component, Security Core and in Validator Component

### LINKED ISSUE #47819: Want to add Urdu Language Translation in Form Component, Security Core and in Validator Component
### Description

I would love to do the translations for the Urdu language in the Symfony Translations.
Urdu is a language that is spoken in Pakistan. And Urdu language is my mother tongue.
Adding this Language in Symfony Translations would help developers from Pakistan.

Thanks,  would love to work on this when it's accepted.

### Example

_No response_

### PR DESCRIPTION
**Pull request by MuhammadAakash**
… in Validator Component for Symfony Translations

| Q             | A
| ------------- | ---
| Branch?       | 6.2 
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | Fix #47819 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Added the Urdu language support in all the translations file. So, that helps those people who speak the Urdu language while working in symfony

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.ur.xlf [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+                <source>This value should be between {{ min }} and {{ max }}.</source>
+                <target> کے درمیان ہونی چاہیے۔ {{ max }} اور {{ min }} یہ ويليو</target>
+            </trans-unit>
+            <trans-unit id="95">
+                <source>This value is not a valid hostname.</source>
+                <target> نہیں ہے۔ hostname یہ ويليو درست </target>
+            </trans-unit>
+            <trans-unit id="96">
+                <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
+                <target> کی ضرب ہونی چاہیے۔ {{ compared_value }} اس مجموعہ میں عناصر کی تعداد </target>
```
Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.ur.xlf]:
```suggestion
                <target>کی ضرب ہونی چاہیے۔ {{ compared_value }} اس مجموعہ میں عناصر کی تعداد</target>
```

It looks like some translations are whitespaces that should be removed. Here is one such example. Can you remove them all?

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.ur.xlf (hunk 1 above)
Comment (MuhammadAakash) [reply src/Symfony/Component/Validator/Resources/translations/validators.ur.xlf]: Sure sir. 
No problem I can fix them and submit an other PR.

Comment (javiereguiluz) [reply src/Symfony/Component/Validator/Resources/translations/validators.ur.xlf]: @MuhammadAakash if possible, please try to make those changes in this PR so we can merge everything at once. Thank you!

Comment (MuhammadAakash) [reply src/Symfony/Component/Validator/Resources/translations/validators.ur.xlf]: @javiereguiluz 
Sure.

Comment (MuhammadAakash) [reply src/Symfony/Component/Validator/Resources/translations/validators.ur.xlf]: Hi, sir @fabpot @javiereguiluz. Hopefully, you are doing great. I have removed all the extra spaces from the files.
Please review.
Thanks.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @MuhammadAakash.

**Comment by MuhammadAakash** [comment]
@fabpot 
It's my pleasure to contribute to Symfony.

**Pull request merged**